lib/src/main/java/uk/gov/di/ipv/core/library/domain/gpg45/domain/CredentialEvidenceItem.java

```diff
@@ -77,7 +77,7 @@ public Gpg45Scores.Evidence getEvidenceScore() {
         return new Gpg45Scores.Evidence(getStrengthScore(), getValidityScore());
     }
 
-    public boolean hasContraIndicators() {
+    public boolean hasContraIndicatorsExcludingFraudA01() {
```

Comment:
I think this should be refactored slightly now. We no longer really need to be hard coding the fraud A01 scenario anymore.

I think this method should go back to its original name, and the logic inside that does the check of it being fraud etc can be removed. This method should just return true if any item has any CI at all

Reply:
So this is used for the VC success check for the audit event in retrieve-cri-credential - I was thinking it should still be considered "success" if it's fraud with only the A01 CI?

Reply:
I think that would be a change in the behaviour from before. Currently for the audit event, if it's fraud with A01 that is marked as not successful. Its only marked as successful for the select-cri bit

Reply:
I think for the audit event the logic we went with originally, was that if any CI is on the VC then it would be marked as not successful.

Reply:
Ah I see what you mean now yeah, I forgot about the flag being set to false there. Will update this then